src/pages/Mypage/UserInfo/UserInfo.style.js

```diff
@@ -1,9 +1,79 @@
 import styled from 'styled-components';
 
 export const CategoryContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
   padding: 50px;
+  height: 80%;
+  width: 60vw;
+`;
+
+export const UserInfoBox = styled.div`
+  display: flex;
+  height: 30%;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
```

Comment:
```suggestion
  display: flex;
  align-items: center;
  justify-content: center;
  height: 30%;
  width: 100%;
```

1차 프로젝트때는 stylelint가 이런 속성의 순서도 일관성있게 linting 해줬는데, 이제는 없습니다.
연관된 속성은 붙여서 사용해보세요.